modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/BatchTestClassGroup.java

```diff
@@ -68,6 +68,9 @@ public long getAverageTestDuration(String testName) {
 				averageTestDuration = testHistory.getAverageDuration();
 			}
 		}
+		else {
+			averageTestDuration = 0;
+		}
```

Comment:
@brittneyq This change seems to conflict with _getDefaultTestDuration(). Shouldn't _getDefaultTestDuration() return 0?